src/core/xds/grpc/xds_client_grpc.cc

```diff
@@ -232,7 +232,7 @@ GetStatsPluginGroupForKeyAndChannelArgs(absl::string_view key,
     return GlobalStatsPluginRegistry::GetStatsPluginsForServer(channel_args);
   }
   grpc_event_engine::experimental::ChannelArgsEndpointConfig endpoint_config(
-      ChannelArgs{});
+      ChannelArgs{channel_args});
```

Comment:
No need for `ChannelArgs{}` around this -- we can just pass `channel_args` directly to the endpoint_config ctor.

Reply:
ofcourse :D